README.md

```diff
@@ -1,8 +1,117 @@
 
 This tool will help implement circularity metrics, quantify and assign a value framework to efforts on re-design, reduction, replacement, reusage, recycling, and lifetime and reliability increases on PV.
 
+
+Documentation
+=============
+
+The calculator follows the following diagram for calculating Mass Flow. Baseline inputs are available in the CEMFC \ baselines folder for US and World past and projected values. Full documentation can be found at [readthedocs](http://CircularEconomy-MassFlowCalculator.readthedocs.io/en/latest/).
+
+<img src="docs/images_wiki/MFC-Diagram.PNG" width="550">
+
+[Input data](https://docs.google.com/spreadsheets/d/1WV54lNAdA2uP6a0g5wMOOE9bu8nbwvnQDgLj3GuGojE/edit?usp=sharing) references include:
+
+Real world installation data from IEA-PVPS reports for 1995 - 2010 (K. Bolcar and K. Ardani, “National Survey Report of PV Power Applications in the United States 2010,” IEA-PVPS, National Survey T1-19:2010, 2010. [Online]. Available: https://iea-pvps.org/national-survey-reports/.)
+
+Projection installation data 2010-2019 from Q4 2019 Wood Mackenzie Power & Renewables/SEIA U.S. Solar Market Insight (A. Perea et al., “U.S. Solar Market Insight: Q4 2019,” SEIA, Wood Mackenzie Power & Renewables, Dec. 2019.)
+
+Installations projected to increase 8.9% compound annual growth rate through 2050 (IRENA, “Future of Solar PV 2019,” IRENA, 2019. Accessed: Apr. 02, 2020. [Online]. Available: https://irena.org/-/media/Files/IRENA/Agency/Publication/2019/Nov/IRENA_Future_of_Solar_PV_2019.pdf.)
+
+Degradation rate (in percentage power loss per year) (D. C. Jordan, S. R. Kurtz, K. VanSant, and J. Newmiller, “Compendium of photovoltaic degradation rates,” Progress in Photovoltaics: Research and Applications, vol. 24, no. 7, pp. 978–989, 2016, doi: 10.1002/pip.2744.)
+
+Glass thickness data (ITRPV, “International Technology Roadmap for Photovoltaic (ITRPV) 2019,” ITRPV, 10th, Oct. 2019.)
+
+Reliability data, i.e. T50 and T90 lifetime in years (D. C. Jordan, B. Marion, C. Deline, T. Barnes, and M. Bolinger, “PV field reliability status—Analysis of 100 000 solar systems,” Progress in Photovoltaics: Research and Applications, vol. n/a, no. n/a, Feb. 2020, doi: 10.1002/pip.3262.)
+
+All manufacturing, recyling, collection efficiencies currently estimates with real world data pending.
+
+
+Installation
+============
+
+CircularEconomy-MassFlowCalculator releases may be installed using the ``pip`` and ``conda`` tools.
+Please see the [Installation page](http://CircularEconomy-MassFlowCalculator.readthedocs.io/en/latest/installation.html) of the documentation for complete instructions.
+
+CircularEconomy-MassFlowCalculator is compatible with Python 3.5 and above.
+
+Install with:
+
+    pip install CEMFC
```

Comment:
@mdeceglie I'm confused about this comment because I included some scripts for it to pip install, and when I tested it in my computer  pip install CEMFC works. Have you tried it on your computer? Am I missing a step?  Please advise.

Reply:
Sorry, I should have been more specific. `pip install CEMFC` works for me, but only if I'm in the source directory. If the project is on pypi.org (e.g. https://pypi.org/project/bifacial-radiance/), then pip will go check pypi.org and download the package automatically no need to separately download the source. 

Here's the official tutorial: https://packaging.python.org/tutorials/packaging-projects/ 

I'd recommend getting it published on pypi once the release is ready.